src/CMakeLists.txt

```diff
@@ -724,6 +740,9 @@ if(BUILD_CLIENT)
 	if (USE_SPATIAL)
 		target_link_libraries(${PROJECT_NAME} ${SPATIAL_LIBRARY})
 	endif()
+	if (USE_OPENSSL)
+		target_link_libraries(${PROJECT_NAME} OpenSSL::Crypto)
+	endif()
```

Comment:
I thought that too, but it doesn't work, I get a linking error. Adding a `PUBLIC` also doesn't work.

Maybe it's because we don't `target_link_libraries` with `EngineCommon`, but just `target_sources` with `$<TARGET_OBJECTS:EngineCommon>`?

Reply:
hmm, maybe